aws_secretsmanager_agent/src/logging.rs

```diff
@@ -26,6 +27,7 @@ const MAX_LOG_ARCHIVE_FILES: u32 = 5;
 const BYTES_PER_MB: u64 = 1024 * 1024;
 const MAX_ALLOWED_LOG_SIZE_IN_MB: u64 = 10;
 const FILE_APPENDER: &str = "FILE_APPENDER";
```

Comment:
These two constants (FILE_APPENDER and CONSOLE_APPENDER) can be moved into the match statement scope, they don't seem to be used anywhere else